src/OneExplorer/OneExplorer.ts

```diff
@@ -893,23 +888,23 @@ input_path=${modelName}.${extName}
 
   getChildren(element?: Node): vscode.ProviderResult<Node[]> {
     if (!element) {
-      element = this.getTree();
+      return this.getTree();
```

Comment:
FOR REVIEWERS
Here, returning this.getTree() is enough. Because the getTree() returns `the first directory node(s)` which are virtually child(ren) node(s) of the ONE Explorer View's root.